resources/js/Pages/Adminland/Hardware/Create.vue

```diff
@@ -73,7 +73,6 @@ input[type=radio] {
                 :label="$t('account.hardware_create_lend_hardware')"
                 :extra-class-upper-div="'mb0 relative'"
                 :required="false"
-                @change="updateStatus($event)"
```

Comment:
This breaks this screen. We can't assign an employee anymore.

![image](https://user-images.githubusercontent.com/61099/113801733-6e00d880-9727-11eb-8c24-9aaef6b56642.png)

Reply:
I saw this error, but it's not this line that is doing it ... 
I still don't know what is broken :(

Reply:
Yeah, fixed!